components/nexus-repository/src/main/java/org/sonatype/nexus/repository/storage/StorageFacet.java

```diff
@@ -53,6 +53,12 @@
   static String P_VERSION = "version";
 
   /**
+   * Registers format specific selector for {@link WritePolicy}. If not set, the {@link
+   * WritePolicySelector#DEFAULT} is used (returns the configured write policy).
```

Comment:
s/is used (returns the configured write policy)/is used which returns the configured write policy/